web-admin/tests/setup/setup.ts

```diff
@@ -232,35 +223,17 @@ setup.describe("global setup", () => {
     ).toBeVisible();
 
     // Wait for the first dashboard to be ready
-    await expect
-      .poll(
-        async () => {
-          await adminPage.reload();
-          const listing = adminPage.getByRole("link", {
-            name: "Programmatic Ads Auction auction_explore",
-          });
-          return listing.textContent();
-        },
-        {
-          // Increased timeout for the 1st dashboard to make sure sources are reconciled.
-          intervals: [10_000, 10_000, 20_000, 20_000, 30_000, 30_000],
-          timeout: 120_000,
-        },
-      )
-      .toContain("Last refreshed");
+    await expect(
+      adminPage.getByRole("link", {
+        name: "Programmatic Ads Auction auction_explore",
+      }),
+    ).toContainText("Last refreshed");
 
-    await expect
-      .poll(
-        async () => {
-          await adminPage.reload();
-          const listing = adminPage.getByRole("link", {
-            name: "Programmatic Ads Bids bids_explore",
-          });
-          return listing.textContent();
-        },
-        { intervals: Array(6).fill(5_000), timeout: 30_000 },
-      )
-      .toContain("Last refreshed");
+    await expect(
+      adminPage.getByRole("link", {
+        name: "Programmatic Ads Bids bids_explore",
+      }),
+    ).toContainText("Last refreshed");
```

Comment:
I think these `expect` assertion timeouts are 5s. That might be a little tight. Can you increase them to 15s? E.g.:
```
).toContainText("Last refreshed", { timeout: 15_000 });
```